templates/index.html

```diff
@@ -27,29 +47,49 @@
     <div class="container">
       <div class="row">
         <h1>What can GeoSAFE do?</h1>
-        <div class="row">
-            <ul>
-                <li>
-                    <b>Perform Analysis</b>
-                    <p>
-                        Analyze a map according to the given layers (exposure and hazard) and generate InaSAFE report
-                    </p>
-                </li>
-                <li>
-                    <b>Analysis List</b>
-                    <p>
-                        Contain saved map analysis for future reference
-                    </p>
-                </li>
-                <li>
-                    <b>Metasearch</b>
-                    <p>
-                        Search based on Catalog Service for the Web (CSW)
-                    </p>
-                </li>
-            </ul>
+
+          <div class="row">
+              <table style="margin: 25px 0px;">
+                  <tr>
+                      <td>
+                          <div id='mySwipeWhat' style='max-width:640px;max-height: 480px;margin:0 auto' class="swipe-what">
+                              <div class='swipe-what-wrap'>
+                                  <div><img src="{% static 'geosafe/img/geosafe-menu.png' %}"/></div>
+                                  <div><img src="{% static 'geosafe/img/analysis-list.png' %}"/></div>
+                                  <div><img src="{% static 'geosafe/img/analysis-result.png' %}"/></div>
+                                  <div><img src="{% static 'geosafe/img/meta-search.png' %}"/></div>
+                              </div>
+                          </div>
+                      </td>
+                      <td>
+                          <ul style="margin: 5px 50px;">
+                              <li>
+                                  <b>Perform Analysis</b>
+                                  <p>
+                                      Analyze a map according to the given layers (exposure and hazard) and generate
+                                      InaSAFE report
+                                  </p>
+                              </li>
+                              <li>
+                                  <b>Analysis List</b>
+                                  <p>
+                                      Contain saved map analysis for future reference
```

Comment:
Browse previously run InaSAFE analysis results and visualize the map and report products for each analysis.